### PR TITLE
Making compatible with Python2.7 and README changes.

### PR DESCRIPTION
**Pull request by shubhamjain0594**
- Small changes for compatibility with different python versions
- Readme changes for better installation instructions.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @shubhamjain0594! I've created [OSPR-2288](https://openedx.atlassian.net/browse/OSPR-2288) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf)     or indicated your institutional affiliation.     If you like, you can add yourself to the [AUTHORS](https://github.com/edx/codejail/blob/master/AUTHORS)      file for this repo, though that isn't required.     Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

**Comment by nedbat** [comment]
@shubhamjain0594 Thanks!  Once you sign the contributor agreement, we can start working on this pull request.
Ideally, you'd make separate pull requests for distinct changes.  This pull request is two separate changes, but they are small, so we can work with this.

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,6 @@ htmlcov
 
 # OS detritus
 *.DS_Store
+
+dist/
+sftp-config.json
```
Comment (nedbat) [inline .gitignore]: This isn't a file that has anything to do with codejail.  You can exclude in a global git ignore if it's something you often end up with in your working trees.

File: codejail/jail.py [hunk 2]
```diff
@@ -147,7 +148,8 @@ def safe_exec(self, code, globals_dict, files=None, python_path=None, slug=None,
                 files.append(pydir)
 
         stdin = json.dumps([code, json_safe(globals_dict)])
-        jailed_code = self.lang.safe_exec_template % {'python_path': '\n'.join(pypath_code)}
+        jailed_code = self.lang.safe_exec_template % {'python_path': '\n'.join(
+            pypath_code)}
```
Comment (nedbat) [inline codejail/jail.py]: Please don't change the formatting of lines you aren't changing. There's no reason to wrap this line like this.

File: codejail/safe_exec.py [hunk 3]
```diff
@@ -54,8 +54,7 @@ def not_safe_exec(
         files=None,
         python_path=None,
         slug=None,
-        extra_files=None
-    ):  # pylint: disable=unused-argument
+        extra_files=None):  # pylint: disable=unused-argument
```
Comment (nedbat) [inline codejail/safe_exec.py]: No need to re-wrap this line.

File: codejail/jail.py (hunk 2 above)
Comment (shubhamjain0594) [reply codejail/jail.py]: I thought we are following PEP8 standards.

**Comment by shubhamjain0594** [comment]
Hi @nedbat , agreement signed  and sent. Suggestions have been acknowledged.

**Comment by nedbat** [comment]
@shubhamjain0594 Thanks, we've received your contributor agreement.

**Comment by nedbat** [comment]
Oops, looks like we still have something to work out about the agreement.

**Comment by mduboseedx** [comment]
@shubhamjain0594 Were you able to complete the agreement process?

**Comment by mduboseedx** [comment]
@shubhamjain0594 I'll close out this PR for now, but feel free to reopen it once the contributor agreement has been finalized. Thanks!

**Comment by edx-webhook** [comment]
@shubhamjain0594 Even though your pull request wasn’t merged, please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/codejail&entry.867055334=https://github.com/edx/codejail/pull/57&entry.1484655318=https://github.com/shubhamjain0594&entry.752974735=2018-03-11+22:27&entry.1917517419=2018-07-31+17:43&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**